controllers/insertCommentsForAnArticleController.js

```diff
@@ -0,0 +1,17 @@
+const {insertCommentsForAnArticleControillers} = require('../models/articlesModel')
+
+const getArticles = async(req,res,next) =>{
+
+   // console.log("Hi")
```

Comment:
It's commented out, but still worth removing at some point because you want this to be a clean version of the code